### PR TITLE
Redux Project - Bookstore, Milestone 4

### PR DESCRIPTION
**Pull request by tanzila-abedin**
I have done the following for this milestone: 
- Implement current books filter in the redux store
- Implement the CategoryFilter component to render a select box with filter categories.
- Keep the filter value in the Redux store
-  Deployed current changes to Heroku

Thank you in advance for the review ⭐

### CODE REVIEW

File: src/components/Categoryfilter.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+const CategoryFilter = ({ options, handleChange }) => (
+  <div>
+    <p> Select Category </p>
+    <Select options={options} onChange={(e) => handleChange(e)} required />
```
Comment (eliasecasta) [inline src/components/Categoryfilter.js:8]: - [x] The "All" option for the category list should also appear, please add it to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @tanzila-abedin, I am Elias, your reviewer for this PR

Well done so far with your project!

You did great in this milestone and only missed a really simple and easy-to-solve mistake, keep it up! 👍 

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please add the "All" option to your filter.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Categoryfilter.js (hunk 1 above)
Comment (tanzila-abedin) [reply src/components/Categoryfilter.js:8]: Hello @e71az 

'All' option do appear on top of  category list . Kindly have a look at the booklist.js file, I added "all" as an option inside useEffect method.
Screenshot from 2021-11-03 00-27-52

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @tanzila-abedin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categoryfilter.js (hunk 1 above)
Comment (eliasecasta) [reply src/components/Categoryfilter.js:8]: Sorry, I just noticed the second bar to assign a category to a book but never saw the first one, good luck in your next milestone tanzila!

**Pull request merged**